libraries/plugins/es_objects/es_objects.cpp

```diff
@@ -47,13 +47,14 @@ class es_objects_plugin_impl
       {  curl = curl_easy_init(); }
       virtual ~es_objects_plugin_impl();
 
-      bool updateDatabase( const vector<object_id_type>& ids , bool isNew);
+      bool indexDatabase( const vector<object_id_type>& ids, std::string action);
+      void RemoveFromDatabase( object_id_type id, std::string index);
```

Comment:
Please start method names with a lowercase letter.

I don't mind camelCase, but usually we use underscores instead.

Reply:
We should have all methods with lowercase and underscore as `camel_case` as part of our code styles. I changed everything to that form in  https://github.com/bitshares/bitshares-core/pull/1271/commits/fc2a9b15b59cc6332a501e7764e4c966c124fc18 and https://github.com/bitshares/bitshares-core/pull/1271/commits/229f215ff1b068e364fd04329d83bdd2de871729

Thanks @pmconrad